js/src/dom/polyfill.js

```diff
@@ -71,11 +71,7 @@ if (!supportScopeQuery) {
 
     const matches = find.call(this, selector)
 
-    if (typeof matches[0] !== 'undefined') {
-      return matches[0]
-    }
-
-    return null
+    return matches[0] ? matches[0] : null
```

Comment:
The old logic checked if `matches[0]` is `undefined`, while new logic checks if `matches[0]` is falsey. *Assuming that change is okay* (and from what I can tell about return types of `find()`, I believe that's true), it's also possible to be DRYer with:
```js
return matches[0] || null // And if you're using the fanciest of ES/JS, you could even do `?? null` which more closely fits the old logic
```

Reply:
We usually prefer the ternary operator than the shortcut one. Also, we don't make use of the nullish operator in the codebase yet.